### PR TITLE
Must do untimely date math client side for nonrated issues

### PR DESCRIPTION
**Pull request by pkarman**
connects #7300 

### Description
Feature tests were missing for eligibility issues on nonrated issues. These left us open to a regression because nonrated issues cannot have their timeliness pre-computed server-side (because they are, by definition, not connected to Rating Issues).

### Acceptance Criteria
- [x] Code compiles correctly
- [x] You can add nonrated issues to an Intake if the decision date was in the last year.

### CODE REVIEW

File: spec/feature/intake/appeal_spec.rb [hunk 1]
```diff
@@ -379,13 +383,28 @@ def check_row(label, text)
...
+    fill_in "Issue category", with: "Active Duty Adjustments"
+    find("#issue-category").send_keys :enter
+    fill_in "Issue description", with: "Another Description for Active Duty Adjustments"
+    fill_in "Decision date", with: "04/19/2016"
+    expect(page).to have_button("Add this issue", disabled: false)
+    safe_click ".add-issue"
+    expect(page).to have_content("6 issues")
+    expect(page).to have_content(
+      "Another Description for Active Duty Adjustments #{Constants.INELIGIBLE_REQUEST_ISSUES.untimely}"
+    )
```
Comment (shanear) [inline spec/feature/intake/appeal_spec.rb]: Do we have a test for the success case? that seems like the scenario that was broken

File: spec/feature/intake/appeal_spec.rb [hunk 2]
```diff
@@ -361,6 +361,10 @@ def check_row(label, text)
     expect(page).to have_button("Add this issue", disabled: false)
     safe_click ".add-issue"
     expect(page).to have_content("2 issues")
+    # this nonrated issue is timely
+    expect(page).to_not have_content(
+      "Description for Active Duty Adjustments #{Constants.INELIGIBLE_REQUEST_ISSUES.untimely}"
```
Comment (pkarman) [inline spec/feature/intake/appeal_spec.rb]: yes the success case test is here

File: spec/feature/intake/higher_level_review_spec.rb [hunk 3]
```diff
@@ -637,6 +637,10 @@ def check_row(label, text)
       expect(page).to have_button("Add this issue", disabled: false)
       safe_click ".add-issue"
       expect(page).to have_content("2 issues")
+      # this nonrated issue is timely
+      expect(page).to_not have_content(
+        "Description for Active Duty Adjustments #{Constants.INELIGIBLE_REQUEST_ISSUES.untimely}"
```
Comment (pkarman) [inline spec/feature/intake/higher_level_review_spec.rb]: another success test

File: spec/feature/intake/supplemental_claim_spec.rb [hunk 4]
```diff
@@ -500,6 +500,8 @@ def check_row(label, text)
       expect(page).to have_button("Add this issue", disabled: false)
       safe_click ".add-issue"
       expect(page).to have_content("2 issues")
+      # SC is always timely
+      expect(page).to_not have_content("Description for Active Duty Adjustments is ineligible because it has a prior")
```
Comment (pkarman) [inline spec/feature/intake/supplemental_claim_spec.rb]: SC should always succeed

**Pull request merged**